catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -498,6 +501,14 @@ public void setIngest(Boolean isIngest) {
         this.isIngest = isIngest;
     }
 
+    public Boolean getAllowEditing() {
```

Comment:
Doesn't appear that way in this file.  I do see the variable names are like that though.